azure/durable_functions/models/DurableEntityContext.py

```diff
@@ -0,0 +1,219 @@
+from typing import Optional, Any, Dict, Tuple, List, Callable
+from azure.functions._durable_functions import _deserialize_custom_object
+from datetime import datetime
+import json
+
+
+class DurableEntityContext:
```

Comment:
Is there a reason we aren't supporting the signal_entity API?

Reply:
I _could_ implement it quickly enough, now that the rest of the entity infrastructure is in place.
But, since `call_entity` already enables you to do the same things as `signal_entity` (just slower), I was hoping to not make it a GA-blocking feature. In other words, I just figured its low priority at this point :)

Reply:
It's now supported